fritz

```diff
@@ -143,24 +150,51 @@ def run(args):
 
     # start up skyportal
     # create kowalski_kowalski network if it does not exist yet
-    p = subprocess.run(["docker", "network", 'create', "kowalski_kowalski"])
-    # docker-compose.skyportal.yaml bind-mounts the fritz-created config.yaml
-    p = subprocess.run(["docker-compose", "-f", 'docker-compose.skyportal.yaml', "up", "-d"], cwd="skyportal")
+    p = subprocess.run(
+        ["docker", "network", 'create', "kowalski_kowalski"],
+        capture_output=True,
+        universal_newlines=True
+    )
+    if p.returncode != 0 and "already exists" not in p.stderr:
+        raise RuntimeError("Failed to create network kowalski_kowalski")
+
+    # docker-compose.skyportal.yaml bind-mounts the fritz-specific config.yaml and db_seed.yaml
+    p = subprocess.run(
+        ["docker-compose", "-f", 'docker-compose.skyportal.yaml', "up", "-d"],
+        cwd="skyportal",
+        check=True
+    )
     if p.returncode != 0:
-        raise RuntimeError("Failed to start skyportal")
+        raise RuntimeError("Failed to start SkyPortal")
 
     if args.init:
-        # init skyportal and load test data
-        p = subprocess.run(["docker", "exec", "-it", "skyportal_web_1", "/bin/bash", "-c",
-                            "source /skyportal_env/bin/activate; make load_demo_data"], cwd="skyportal")
-        if p.returncode != 0:
-            raise RuntimeError("Failed to init skyportal and load test data")
+        # init skyportal and load seed data
+        mi, max_retires = 1, 3
+        while mi <= max_retires:
+            p = subprocess.run(
+                [
+                    "docker", "exec", "-it", "skyportal_web_1", "/bin/bash", "-c",
+                    # wait for SP to be up and running:
+                    # 'while [[ $(ps -ef | grep "make" | wc -l) -lt 4 ]]; '
+                    # 'do echo "waiting for SkyPortal to come up"; sleep 5; done;'
+                    "source /skyportal_env/bin/activate; make db_init; make load_seed_data"
```

Comment:
```suggestion
                    "source /skyportal_env/bin/activate; make db_clear; make db_init; make load_seed_data"
```
I found that I ran into load_seed_data errors due to the data already existing in the database when trying to run init multiple times.

Reply:
Yea, but this is a feature, not a bug AFAIK as you're only supposed to init once. Unfortunately, we cannot do `make db_clear` as it requires sudo privileges, which is explicitly avoided, again AFAIK